[OGUI-1711] Layout for tokenisation application

mroki58 opened the pull request:
The most important changes:
- header with changing label using context API
- sidebar for whole app
- navbar element for different pages.


#### I have JIRA issue created
- [x] branch and/or PR name(s) includes JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [ ] FLP integration tests were ran successful

mroki58 committed:
Layout for application created, documentation added.

The most important changes:
- header with changing label using context API
- sidebar for whole app
- navbar element for different pages.

graduta reviewed (changes requested):
Please have a look at the formating of the files. You have a few places where:
- you add lots of new lines with no need
- you are missing end of file new line in some files
- you have unused variables such as `Box` in `overview.tsx` and other occurences
- please use spacing of `2` instead of `4` (e.g. root.tsx). I apologise if by mistake I added files int he initial project with spaces of 4. We should address this by adding eslint checker to the project's pipeline
- please use as a standard positioning of external libraries (such as react and material) imports at the top and relative path imports afterwards
- please use classes defined in the CSS files for elements instead of direct styling in the component (e.g. layout.tsx)

mroki58 committed:
Merge branch 'dev' into feature/TKN/OGUI-1711/implement-overall-layout

mroki58 committed:
- Added eslint configuration with established assumptions

- Added needed packages with eslint for js and ts

- move style to css files - kept in style directory

- changes in structure of code without affecting any appearance

graduta committed:
Merge branch 'dev' into feature/TKN/OGUI-1711/implement-overall-layout
